src/app/modules/pages/components/sign-up-form/sign-up-form/sign-up-form.component.ts

```diff
@@ -42,29 +40,33 @@ export class SignUpFormComponent {
 		return this.signUpForm.controls['confirmPassword'] as FormControl;
 	}
 
-	public send() {
+	public async onSubmit() {
 		if (this.signUpForm.invalid) {
+			this.notifier.notify('error', 'Somethings wrong :(');
```

Comment:
no need to notify here. Only on back-end error, because user will see error on the form